FAQ.md

```diff
@@ -0,0 +1,24 @@
+## Frequently Asked Questions
```

Comment:
Hey @mollyIV, first of all, thanks for adding this content to the repository. I'd like to suggest moving this to a new page in the documentation website. We can place it in this directory https://github.com/tuist/tuist/tree/master/website/markdown/docs/usage alongside the other usage documentation pages. You can have a look at how the others are defined to follow the same structure.